documentation/src/docs/asciidoc/index.adoc

```diff
@@ -15,6 +15,8 @@ ifdef::backend-pdf[:imagesdir: {outdir}/{imagesdir}]
 :javadoc-root:                      link:../api
 //
 :snapshot-repo:                     https://oss.sonatype.org/content/repositories/snapshots
+:stackoverflow:                     https://stackoverflow.com/questions/tagged/junit5
+:gitter-chat:                       https://gitter.im/junit-team/junit5
```

Comment:
For consistency, these new links should be placed (alphabetically) at the bottom alongside the links to AssertJ, etc.